.gitlab-ci.yml

```diff
@@ -1077,6 +1077,8 @@ latest_dsd_push:
 deploy_cloudfront_invalidate:
   stage: deploy_invalidate
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/deploy:latest
+  before_script:
```

Comment:
Could we add a comment explaining why we do this ?

Reply:
Sure, this confused me pretty hard so it's probably worth it